distribution/openrct2.d.ts

```diff
@@ -1068,12 +1068,24 @@ declare global {
         /**
          * The progress on the current track piece, in steps.
          */
-        trackProgress: number;
+        readonly trackProgress: number;
+		
+        /**
+         * The currently projected remaining distance the car will travel.
+         */
+        readonly remainingDistance: number;
 
         /**
          * List of peep IDs ordered by seat.
          */
         peeps: (number | null)[];
+				
+        /**
+         * Moves the vehicle forward or backwards along the track, relative to its current 
+		 * position. A single visible step is about 8.000 to 14.000 in distance depending 
+		 * on the direction its moving in.
+         */
```

Comment:
The formatting is a bit strange here.

Reply:
The formatting was fixed with commit 1cfaf49 right? Or did I miss something?

Reply:
Yes, my browser probably hadn't updated to show the diff with that commit.